appengine_flexible/memcache/memcache.go

```diff
@@ -3,6 +3,7 @@
 // license that can be found in the LICENSE file.
 
 // Sample memcache demonstrates use of a memcached client from App Engine flexible environment.
+// [START gae_flex_redislabs_memcache]
```

Comment:
Same as above, move this to above the line starting with `// Sample` and put a newline in between.